docs/lms-openapi.yaml

```diff
@@ -2,17 +2,19 @@ swagger: '2.0'
 info:
   title: Open edX API
   description: APIs for access to Open edX information
+  contact:
```

Comment:
I was trying to remember why in the world we decided to commit this generated file to source control. I looks like when I approved it originally, it was[ contingent on coming up with some automatic way to keep it in sync with reality](https://github.com/openedx/edx-platform/pull/21207#discussion_r316678430), which, oops, we never did.

My gut feeling back then was that we shouldn't commit it without a syncing strategy. I still feel that way, especially now that it's part of `make docs` -- it'll be annoying to have `make docs` dirty the git state.

If you want to let this be for now, that's fine. But if you'd like to git-rm this file and put it in .gitignore, you'd have my 👍🏻

Reply:
We use this file as a part of the docs now, it generates RST docs based on the openapi schema. I also think it's being used operationally in a few places so it's removal is a bit more complex.  I generally agree with you though, I don't like how it's being managed currently.  There are some improvements we can make here as we improve the edx-platform API posture.